### ISSUE TITLE
Unable to start BBB call in rocket chat

### ISSUE THREAD

**Issue by netlabs** [RocketChat/Rocket.Chat#13945]
When we try to call we get this page and no response afterward. Please help

![image](https://user-images.githubusercontent.com/45689540/55214038-67c3f180-521b-11e9-9232-dd204b847ea5.png)

**Comment by reetp** [comment]
If you believe there is an Issue with Rocket please read this carefully:
https://rocket.chat/docs/contributing/reporting-issues

Then fill in the the bug template and paste the information here:
https://github.com/RocketChat/Rocket.Chat/issues/new?template=bug_report.md

Server setup and logs are required. No one can help you without this information

**Comment by Shaltz** [comment]
Same problem here... servers are setup, working fine by themselves, but can't get them to work together... the bbb video Chat option from rc is present but when I click it, nothing happens beside a 500 error in the dev console
`r {isClientSafe: true, error: 500, reason: "Internal server error", details: undefined, message: "Internal server error [500]", …}
details: undefined
error: 500
errorType: "Meteor.Error"
isClientSafe: true
message: "Internal server error [500]"
reason: "Internal server error"
stack: "Error: Internal server error [500]↵    at w._livedata_result (...`


any idea ?

**Comment by slowflyer** [comment]
I got this error fixed, when realized that web hooks are needed to installed for BBB

After running 

sudo apt-get install bbb-webhooks

on my BBB installation, I was able to start a Video Conference in Rocket Chat

**Comment by karmapandya** [comment]
@slowflyer Can you share your config settings / screenshot (for rocket chat) ? I can't seem to make it work (with webhooks installed)

Still getting "500 internal server error"

Thanks

**Comment by corrigac** [comment]
- similar issue, but logs indicating 'Error: url must be absolute and start with http:// or https://' so redoing config

**Comment by corrigac** [comment]
- Setup page for BBB chat is misleading 
Domain:   isn't actually the domain, its more like a root url -  https://docs.bigbluebutton.org/dev/webhooks.html#WebHooks_ resource URL component before /bigbluebutton/api/hooks/.....  including protocol indicator and NOT trailing slash

**Comment by karmapandya** [comment]
Yeah. I could not figure out the issue. I always ended up using jitsi for video conferences, so I gave up on BBB. BBB also needs a higher server configuration compared to Jitsi. 

It is way simpler to integrate standalone jitsi meet server.  It meets my requirements. 

There should be simpler documentation for setup. BBB server setup is simple but webhooks documentation is not great.

**Comment by karmapandya** [comment]
Let me know if I can help with anything.

**Comment by einhirn** [comment]
So to sum this up:

- Change the label from "Domain" to "BBB-Webhooks Base URL" or something
- Give an example for a working Base URL
- Inform the user that he needs BBB-webhooks installed on the BBB server

**Comment by slowflyer** [comment]
> @slowflyer Can you share your config settings / screenshot (for rocket chat) ? I can't seem to make it work (with webhooks installed)
> 
> Still getting "500 internal server error"
> 
> Thanks

![Clipboard01](https://user-images.githubusercontent.com/4402834/77438369-027eee00-6de6-11ea-8c81-7d91727b1bdb.jpg)
Here's a screenshot of my settings ... nothing very special

**Comment by einhirn** [comment]
@karmapandya Can you please share a log excerpt from around the Error 500 you are getting? This would reduce the guesswork...

**Comment by karmapandya** [comment]
Hey @einhirn ,

Sorry for the late reply. Unfortunately, I have shut down the server and deleted the data. I don't have a backup of the logs.

**Comment by runawaydevil** [comment]
Sorry for the stupid question but I'm really noob. Well, I did the setup in rocketchat of BBB.

The first thing I did was: bbb-conf --secret
So I get my URL and secret. Put on rocket chat and didn't work.
So I installed webhooks, and take the API-MATE with token and put in the API Mate website, what gave many links to me.
My question is simple, which one of these links should I put on rocket chat to make a call?
What it should looks like? Thanks

**Comment by rasos** [comment]
@runawaydevil you do not need API-MATE links. Just make sure you have installed on your BBB server webhooks.

```
sudo apt-get install bbb-webhooks
bbb-conf --secret
```
Then paste the BBB URL without ending slash and the secret in the settings dialog of RC.

**Comment by borishinzer** [comment]
After a bit of fiddling around, I was able to get the BBB integration runnning.
It was important that the "domain" was actually ment as URL of our BBB - **without** the traling slash an no path behind it.

Example:
https://bbb.web-vision.de/ - not working
https://bbb.web-vision.de/bigbluebutton/ - not working, like show from bbb-conf --secret
https://bbb.web-vision.de/b/ - not working, as this was our default path from the Greenlight GUI
https://bbb.web-vision.de - **working**
